src/lib/hooks/swap/useSwapInfo.tsx

```diff
@@ -86,9 +88,16 @@ function useComputeSwapInfo(): SwapInfo {
     [trade.trade?.inputAmount, trade.trade?.outputAmount]
   )
 
-  // TODO(ianlapham): Fix swap slippage tolerance
-  // const allowedSlippage = useSwapSlippageTolerance(trade.trade ?? undefined)
-  const allowedSlippage = useMemo(() => new Percent(100), [])
+  /**
```

Comment:
`/**` is for doc comments. This should use `/*`.
Also, multi-line comments should start with ` * ` - usually your formatter will do this for you.